packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -93,9 +93,8 @@ function validateOptions (options) {
   }
 
   // validate protocol
-  // @todo this validation should not be conditional once the protocol field is mandatory
-  if (options.protocol && !Object.keys(PROTOCOLS).some((key) => PROTOCOLS[key] === options.protocol)) {
-    throw new ValidationError('unsupported protocol specified')
+  if (options.protocol == null || !Object.keys(PROTOCOLS).some((key) => PROTOCOLS[key] === options.protocol)) {
+    throw new ValidationError('please specify a valid protocol')
```

Comment:
Same here, it should be listed somewhere as a breaking change so we don't forget about it (or land as a separate commit)

Reply:
I'll update the PR's description with breaking changes, then after release we can manually update the description in Releases with the breaking changes, e.g. edit https://github.com/transloadit/uppy/releases/tag/uppy%402.11.0

What do you think about that? Personally when I'm looking for breaking changes in a project on github I usually go to the releases page and find the breaking changes under the major release.

Reply:
The upside of splitting into several commits is the release process will list them automatically, keeping everything in one commit forces us to remember to manually amend the changelog. I would personally prefer if you split it into several commits (and maybe several PRs).